pkg/app/piped/executor/scriptrun/scriptrun.go

```diff
@@ -136,22 +136,26 @@ type ContextInfo struct {
 	ApplicationName     string            `json:"applicationName,omitempty"`
 	TriggeredAt         int64             `json:"triggeredAt,omitempty"`
 	TriggeredCommitHash string            `json:"triggeredCommitHash,omitempty"`
+	TriggeredCommander  string            `json:"triggeredCommander,omitempty"`
 	RepositoryURL       string            `json:"repositoryURL,omitempty"`
 	Summary             string            `json:"summary,omitempty"`
 	Labels              map[string]string `json:"labels,omitempty"`
+	IsRollback          bool              `json:"isRollback,omitempty"`
 }
 
 // NewContextInfo creates a new ContextInfo from the given deployment.
-func NewContextInfo(d *model.Deployment) *ContextInfo {
+func NewContextInfo(d *model.Deployment, isRollback bool) *ContextInfo {
 	return &ContextInfo{
 		DeploymentID:        d.Id,
 		ApplicationID:       d.ApplicationId,
 		ApplicationName:     d.ApplicationName,
 		TriggeredAt:         d.Trigger.Timestamp,
 		TriggeredCommitHash: d.Trigger.Commit.Hash,
+		TriggeredCommander:  d.Trigger.Commander,
 		RepositoryURL:       d.GitPath.Repo.Remote,
 		Summary:             d.Summary,
 		Labels:              d.Labels,
+		IsRollback:          isRollback,
```

Comment:
I fixed to use `isRollback` flag instead of `d.Status`.

That's because `d.Status` is ALWAYS `PLANNED` since scheduler does not update it...